app/code/Magento/CustomerGraphQl/Model/Resolver/Customer/CustomerDataProvider.php

```diff
@@ -42,11 +67,32 @@ class CustomerDataProvider
     public function __construct(
         CustomerRepositoryInterface $customerRepository,
         ServiceOutputProcessor $serviceOutputProcessor,
-        SerializerInterface $jsonSerializer
+        SerializerInterface $jsonSerializer,
+        SubscriberFactory $subscriberFactory,
+        CustomerRegistry $customerRegistry,
+        Encryptor $encryptor,
+        StoreResolverInterface $storeResolver
     ) {
         $this->customerRepository = $customerRepository;
         $this->serviceOutputProcessor = $serviceOutputProcessor;
         $this->jsonSerializer = $jsonSerializer;
+        $this->subscriberFactory = $subscriberFactory;
+        $this->customerRegistry = $customerRegistry;
+        $this->encryptor = $encryptor;
+        $this->storeResolver = $storeResolver;
+    }
+
+    /**
+     * Load customer object
+     *
+     * @param int $customerId
+     * @return CustomerInterface
+     * @throws LocalizedException
+     * @throws NoSuchEntityException
+     */
```

Comment:
Why do we need mathod-wrapper over Repository?

Reply:
Good catch, I planned to use it in another location, but currently it not needed anymore